src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -34,48 +34,36 @@ def __init__(self, table: Table):
 
     def box_plots(self) -> Image:
         """
-        Plot a boxplot for every numerical column.
+        Create a box plot for the values in the column. This is only possible for numeric columns.
```

Comment:
The previous documentation fit better. You say "This is only possible for numeric columns.", so as a user I'd wonder how I could control for which columns box plots should be created. However, this decision happens automatically.

```suggestion
        Create a box plot for every numerical column.
```

Reply:
We reverted the change